src/main/kotlin/additioncalculator/domain/NumberSplit.kt

```diff
@@ -0,0 +1,33 @@
+package additioncalculator.domain
+
+import java.lang.RuntimeException
+
+class NumberSplit(private val text: String?) {
+    fun splitNumberResult(): Numbers {
+        if(text.isNullOrEmpty()) return Numbers(listOf())
+        if(text.toIntOrNull() != null && text.toIntOrNull()!! < NUMBER_MINIMUM)
+            throw RuntimeException(INVALID_NEGATIVE_MESSAGE)
+        if(text.toIntOrNull() != null ) return Numbers(listOf(text.toInt()))
```

Comment:
```suggestion
               if (text.isNullOrEmpty()) {
            return Numbers(listOf())
        }
        if (text.toIntOrNull() != null && text.toIntOrNull()!! < NUMBER_MINIMUM) {
            throw RuntimeException(INVALID_NEGATIVE_MESSAGE)
        }
        if (text.toIntOrNull() != null) {
            return Numbers(listOf(text.toInt()))
        }
```

저는 하나의 라인으로 작성하는 것보다 중괄호를 명시해주는게 가독성이 좋아서 선호하는 편인데요!! 창민님은 어떻게 코드를 작성하시는 편이신가요?

Reply:
> 중괄호를 명시해주는게 가독성이 좋아서 선호하는 편인데요!!

저도 선호님과 완전히 같습니다!

허나, 미션에 주어진 룰과 (함수(또는 메서드)의 길이가 10라인을 넘어가지 않도록 구현한다) 

강의를 들으면서 코틀린 코드는 간결해야한다는 점이 떠올라 라인 수를 줄이기 위해 의도적으로 중괄호를 지웠던 것인데요ㅠ

선호님 리뷰를 보고 라인수를 줄이기 위해 if문의 중괄호를 지우는 것을 의식하기 보다는, 

역할에 맞춰 함수를 잘 분리하도록 의식하는 코드를 작성하는게 협업과 유지 보수성을 위해 더 올바른 방향일 것이라는 생각이 들어 원래 선호하던대로 중괄호를 모두 표현하는 것으로 수정하였습니다!

리뷰를 보고 리팩토링을 거치면서 중괄호가 있음에도 (비교적)충분히 간결한 코드라도 판단이 들었기도 합니다